packages/cactus-plugin-ledger-connector-quorum/src/main/typescript/plugin-ledger-connector-quorum.ts

```diff
@@ -654,4 +711,67 @@ export class PluginLedgerConnectorQuorum
     }
     return this.runDeploy(req);
   }
+
+  // Low level function to call any method from web3.eth
+  // Should be used only if given functionality is not already covered by another endpoint.
+  public async invokeRawWeb3EthMethod(
+    args: InvokeRawWeb3EthMethodV1Request,
+  ): Promise<any> {
+    this.log.debug("invokeRawWeb3EthMethod input:", JSON.stringify(args));
+
+    Checks.nonBlankString(
+      args.methodName,
+      "web3.eth method string must not be empty",
+    );
+
+    const looseWeb3Eth = this.web3.eth as any;
```

Comment:
@outSH Use the `isSafeToCallContractMethod` here as well and then no need to cast into any. Once that's done, I can dismiss the warning (if still present) as a false-positive. If you want more comprehensive checks within the `isSafeToCallContractMethod` function, feel free to add more code in there.

Reply:
@petermetz `isSafeToCallContractMethod` is meant to work with `Contract` instances only (i.e. it checks `methods` member field), and in `invokeRawWeb3EthMethod` I check existance of a method on a `web3.Eth` object. Also, casting to any is necessary here in order to get to a method with string indexing later on. It's not necessary with contracts, since `contract.methods` is already type of `any` :) `web3.Eth` instance that is used here doesn't define string indexing, so without a cast it will report an error.

I've noticed there's duplicated logic of checking method on object though, so I've extracted new function - `isSafeToCallObjectMethod`, that is used by `invokeRawWeb3EthMethod` and `isSafeToCallContractMethod`, so the duplication is eliminated.

Reply:
@outSH Ah, my bad, sorry. Should've been reading the code more carefully. Cool, thanks for the fix, I'll dismiss it as a false positive then and we are good to go.